### DISCUSSION TITLE
Is it possible to reimport old data from MySQL?

### DISCUSSION THREAD

**Discussion by blake-tisbury** [plan-player-analytics/Plan#2522] [Q&A]
I've recently switched hardware and brought my MySQL DB over. However, Plan now only shows data since the switch in the home page tab. (Where you can see a graph over the year). It will still show things like the max peak that was before the switch. Is there any way for it to see the new data and accept it in the main graph view?

**Comment by AuroraLS3** [comment]
Move data to from SQLite to MySQL https://github.com/plan-player-analytics/Plan/wiki/Database-management#moving-data-from-sqlite-to-mysql
